packages/3d-web-client-core/src/character/CharacterManager.ts

```diff
@@ -55,12 +55,14 @@ export class CharacterManager {
     private readonly sendUpdate: (update: CharacterState) => void,
   ) {
     this.group = new Group();
+    setInterval(() => this.update.bind(this), 3000);
   }
 
   public spawnCharacter(
```

Comment:
Split this into `spawnLocalCharacter` and `spawnRemoteCharacter` to avoid having a large binary condition within the arguments and the body